include/config.h

```diff
@@ -25,14 +25,14 @@
 // Use this switch to choose a handler for pretty printing.
 // NOTE: mini_printf supports a custom pretty printing formatter to display preproc encoded strings. (%S)
 //       some libc distributions (especially dkp arm-libc) will fail to link pretty printing.
-#define PRETTY_PRINT_HANDLER (PRETTY_PRINT_OFF)
+#define PRETTY_PRINT_HANDLER (PRETTY_PRINT_MINI_PRINTF)
 
 // Use this switch to choose a handler for printf output.
 // NOTE: These will only work on the respective emulators and should not be used in a productive environment.
 //       Some emulators or real hardware might (and is allowed to) crash if they are used.
 //       AGB_PRINT is supported on respective debug units.
```

Comment:
do the comments about this two defines need changed? looks like this line mentions AGB_PRINT specifically

Reply:
I mean... this line is basically self explanatory. If using mGBA emulator you need to set it to MGBA_PRINT.
It's the same with pokeemerald decomp.
But of course this could be added in the comment.

Reply:
Is fine with me then, go ahead and merge when you want